src/routes/index.js

```diff
@@ -80,7 +81,8 @@ const AllRoutes = (props) => (
           element={<ResetPassword {...props} />}
         />
         <Route exact path="/dashboard/*" element={<DashboardPreview />}>
-          <Route exact path="" element={<Navigate to="trips" />} />
+          <Route path="readone/:id/:role" element={<ReadOneTrip />} />
```

Comment:
Change this route, It doesn't follow a naming convention.  Also, don't pass role in the params